data-model/SDP-sample-records/ms-object_sample1.json

```diff
@@ -0,0 +1,85 @@
+{
+  "id": "ark:/21198/z1x07bdf",
+  "type": "Manuscript Object",
+  "shelfmark": "Sinai Syriac 2A",
+  "collection": "Old Collection",
+  "repository": "https://sinaimanuscripts.library.ucla.edu/place/1",
+  "form": "Codex",
+  "binding": {
+    "exists": true,
+    "type": "",
+    "note": ["Relatively recent; leather on wood showing signs of repair; metal studs (some missing) on front and back and along edges; remnants of clasps (four total, two for edge and one each at top and bottom)"],
+    "features": []
+  },
+  "extent": ["f. 1-180"],
```

Comment:
I'm not sure what to do with extent, weight, and dimensions. At ms object level, do we need just a number of leaves? 

Maybe we define a general 'format' field (following dcterms) that can take various types? Then we can use this for no. of leaves, weight, binding and object dimensions, text block dimensions, etc. as needed.

Alternatively we have a 'format' field that has sub-fields for extent, weight, and an array of dimensions?

Reply:
I think there's no need to wrap these up: we can just use as-is in SMDL for ms object:

- extent: 274 ff.
- weight: xx g
- dimensions: 328 x 255 x 97 mm (I also don't think we need a type since we can interpret from the context)

Reply:
Also add a foliation, which can have both a standardized, regex-parsable form and a more free-form 'notes' field.

We will define a standard way of structuring these standardized strings. Will need to consider how to treat front and end leaves (will they be separately numbered?)

Reply:
dimensions should become an array of strings

Reply:
Adopt definition of the dimension format from the on-site photographers